R/steps-rcmdcheck.R

```diff
@@ -81,11 +81,13 @@ RCMDcheck <- R6Class(
 #' @param warnings_are_errors,notes_are_errors `[flag]`\cr
 #'   Deprecated, use `error_on`.
 #' @param args `[character]`\cr
-#'   Passed to `rcmdcheck::rcmdcheck()`, default:
-#'   `c("--no-manual", "--as-cran")`.
+#'   Passed to `rcmdcheck::rcmdcheck()`.\cr
+#'   Default for Travis and local runs: `"--force"`.\cr
```

Comment:
Please double-check the documentation here. Have you considered moving it closer to the code?